plan/plan.go

```diff
@@ -75,20 +75,20 @@ func newPlanTable() planTable { //TODO: make resolver configurable
 // current corresponds to the record currently occupying dns name on the dns provider
 // candidates corresponds to the list of records which would like to have this dnsName
 type planTableRow struct {
-	current    *endpoint.Endpoint
+	currents   []*endpoint.Endpoint
```

Comment:
What is the reason for changing plan.go?

Reply:
The reason is because the existing logic plan.go addCurrent() is dnsname as the key, a single Current to match Candidates, the change is to support the processing logic of a single dnsname supporting multiple records.